lib/idseq-dag/idseq_dag/steps/prepare_taxon_fasta.py

```diff
@@ -25,7 +26,7 @@ def run(self):
             local_basename = f"{hit_type}_{os.path.basename(output_file)}.fasta"
             bucket, key = s3.split_identifiers(s3_file)
             local_file = os.path.join(self.output_dir_local, local_basename)
-            s3.fetch_byterange(first_byte, last_byte, bucket, key, local_file)
+            download_chunks_to_file(bucket, key, local_file, [first_byte], [last_byte - first_byte])
```

Comment:
I just added this new function for another use case so we can just use it here instead of relying on the old function which used the AWS CLI.